charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -203,12 +206,15 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
+          {{- with .Values.controller.volumeMounts }}
+          {{- toYaml . | nindent 12 }}
+          {{- end }}
```

Comment:
Is there a reason the sidecar containers also need access to these mounts? If not, could you remove them. If yes, could you separate out a config value per-sidecar.

Reply:
Wasn't necessary/